bindings/nodejs/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "target-feature=+crt-static"]
```

Comment:
Is that new? What's that for?

Reply:
No, for windows https://github.com/iotaledger/wallet.rs/pull/1565